airflow-core/src/airflow/jobs/scheduler_job_runner.py

```diff
@@ -515,6 +588,12 @@ def _executable_task_instances_to_queued(self, max_tis: int, session: Session) -
                 locked_query = with_row_locks(query, of=TI, session=session, skip_locked=True)
                 task_instances_to_examine: list[TI] = list(session.scalars(locked_query).all())
```

Comment:
```suggestion
                task_instances_to_examine = session.scalars(locked_query).all()
```

Not really related, but let’s fix this potentially wasteful line